### DISCUSSION TITLE
Layout should not shift on modal open

### DISCUSSION THREAD

**Discussion by GuilhermeAmado** [heroui-inc/heroui#339] [Feedback]
When a modal is open the page scrollbar goes missing and that makes the layout shift a few pixels.
You can see this behavior on the docs page: 
https://nextui.org/docs/components/modal

**Comment by Philip1435** [comment]
Same issue. This creates stumbling user experience. Should there be a fix for this?

**Comment by adamhp** [comment]
+1. It's a bit jarring.

**Comment by MarcusDelvecchio** [comment]
same issue, anyone come up with a fix?

**Comment by London804** [comment]
> When a modal is open the page scrollbar goes missing and that makes the layout shift a few pixels. You can see this behavior on the docs page: https://nextui.org/docs/components/modal

I don't see this behavior happening on Mac in Chrome, Safari or Firefox. Is it safe to assume you are observing this behavior on Windows? What browser/s?

**Reply by everurstruly** [reply]
I use  "**Microsoft Edge - Version 129.0.2792.65 (Official build) (64-bit)**"

**Comment by newguy123-creator** [comment]
I see this weird behaviour on Windows with Firefox and also Chrome
